pages/docs/tooling/tooling-aste.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Artificial Solver Testing Environment (ASTE)
+permalink: tooling-aste.html
+keywords: tooling, aste
+---
+
+## Motivation
+
+ASTE is a tool which can be used to understand the dynamics of preCICE, how the data is goes in and out. It consist of C++ and Python modules.
```

Comment:
```suggestion
ASTE is a tool which can be used to understand the dynamics of preCICE, how the data goes in and out. It consist of C++ and Python modules.
```
I don't like this motivation. Content-wise, we should mention that aste is:
- a lightweight wrapper around preCICE  
- allows to investigate different mapping setups in terms of runtime and accuracy
- allows to investigate different mapping data
- allows to replace individual participants by aste for development purposes